### ISSUE TITLE
Linux : All cores running at lightspeed. Unplayable :(

### ISSUE THREAD

**Issue by 7oxicshadow** [libretro/RetroArch#5224]
## Description

I have just built the latest git revision (6506b71776550b0b5a7c376cee8c7c66313c8761) and ever since all cores run crazy fast. Completely unplayable. If I enable vsync it becomes playable but the audio is terrible.

### Expected behavior

Any / all cores to be at playable speeds

### Actual behavior
All cores run at lightspeed

### Steps to reproduce the bug

1. Build the latest git revision
2. Video set to opengl
3. Load any rom (tested saturn/psx/n64) directly from commandline (no gui interaction)

### Bisect Results
My previous build that was working was "Fri  7 Jul 22:47:46 BST 2017"

### Version/Commit
You can find this information under Information/System Information

- RetroArch: [6506b71]

### Environment information

Fedora 25
GCC 6.3.1
i7 6700k (@ 4.5GHz)
16GB Ram
GTX1070 (Prop 384.59)

**Comment by RobLoach** [comment]
What's in your retroarch.cfg? pastebin?

**Comment by 7oxicshadow** [comment]
https://pastebin.com/EGSKirNq

Thanks

**Comment by RobLoach** [comment]
`input_toggle_fast_forward` is set to "enter". It probably toggles fast forward when you start the game.  Change "enter" to "nul" to just disable it.

**Comment by RobLoach** [comment]
Change it to p, try pressing P during gameplay. It's a toggle, so it is either off or on.

**Comment by 7oxicshadow** [comment]
I tried assigning the option to 'P' but it had no effect when pressing it during gameplay.

In the end I have deleted the retroarch.cfg and retroarch-core-options.cfg to allow retroarch to recreate from scratch with default options.

This has got Retroarch working again :) Sorry for the false alarm.

Note: I have tried comparing the old retroarch config and new and there was only one entry that caught my eye. The audio in my old config was set to 'alsa' whereas the new config is set to use pulse. Very recently i modified a pulse config file to use tsched=0 in order to try and eliminate some crackle issues.

At a guess i suspect the audio timings may have an indirect link to the render timings.

Either way I will close this issue as resolved.

Thanks for your time
